<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto Thin</family>
    <pointsize>24</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="pages">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/main/background.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_project">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>0</y>
       <width>991</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Impact</family>
       <pointsize>72</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#343fa1;&quot;&gt;КАРТОТЕКА ДРУЗЕЙ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_users">
     <property name="geometry">
      <rect>
       <x>1410</x>
       <y>340</y>
       <width>441</width>
       <height>651</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_remove_everyone_info">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>860</y>
       <width>541</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>Узнать подробнее</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>400</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="users_all">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto Thin</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_add_everyone">
        <property name="text">
         <string>Добавить всех</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_remove_everyone">
        <property name="text">
         <string>Удалить всех</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_selected">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>220</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Здесь будут показываться выбранные пользователи&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="user_info">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>790</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Выбери и посмотри информацию о аккаунте&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_all">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Thin</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;О ком хочешь узнать?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>background</zorder>
    <zorder>name_project</zorder>
    <zorder>list_users</zorder>
    <zorder>pushButton_remove_everyone_info</zorder>
    <zorder>verticalLayoutWidget</zorder>
    <zorder>user_info</zorder>
    <zorder>label_all</zorder>
    <zorder>label_selected</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
 </resources>
 <connections/>
</ui>
